robot_skills/src/robot_skills/hero.py

```diff
@@ -20,10 +22,13 @@ def __init__(self, wait_services=False):
         self.add_body_part('base', base.Base(self.robot_name, self.tf_listener))
         self.add_body_part('torso', torso.Torso(self.robot_name, self.tf_listener, self.get_joint_states))
 
-        self.add_arm_part(
-            'leftArm',
-            arms.ForceSensingArm(self.robot_name, self.tf_listener, self.get_joint_states, "left")
-        )
+        # add hero's arm
+        hero_arm = arms.Arm(self.robot_name, self.tf_listener, self.get_joint_states, "left")
+        hero_arm.add_part('force_sensor', force_sensor.ForceSensor(self.robot_name, self.tf_listener, "/" + self.robot_name + "/wrist_wrench/raw"))
+        hero_arm.add_part('gripper', gripper.ParrallelGripper(self.robot_name, self.tf_listener, 'left'))
+        hero_arm.add_part('handover_detector', handover_detector.HandoverDetector(self.robot_name, self.tf_listener, 'left'))
+
+        self.add_arm_part('leftArm', hero_arm)
```

Comment:
Why still the left?

Reply:
because there are still places in our code which require a left arm. And I don't wanna make this PR bigger than it already is.